pkg/isoeditor/stream_test.go

```diff
@@ -68,7 +68,7 @@ var _ = Describe("NewRHCOSStreamReader", func() {
 
```

Comment:
Any reason not to add a test case for the karg case here?

Reply:
Added